src/Kuzzle.js

```diff
@@ -347,38 +386,55 @@ class Kuzzle extends KuzzleEventEmitter {
       request.jwt = this.jwt;
     }
 
-    return this.network.query(request, options);
+    let queuable = true;
+    if (options && options.queuable === false) {
+      queuable = false;
+    }
+
+    if (this.queueFilter) {
+      queuable = queuable && this.queueFilter(request);
+    }
+
+    if (this.queuing && queuable) {
```

Comment:
This means that if the SDK instance is queuing, but the request is not queuable, it is sent to the network.
I'm not sure that this is intended, I would rather think that the request is to be discarded.

And, if this is the case, then maybe the `discarded` event should be triggered.

Reply:
:+1: